docs/readthedocs/source/doc/UserGuide/python.md

```diff
@@ -1,6 +1,9 @@
 # Python User Guide
 
 ---
+Supported Platforms: Linux and macOS.<br>
+ _**Note:** Windows is not supported._
```

Comment:
currently not supported

Reply:
Weird to separate into two lines.
Change to a single line?
```
Supported Platforms: Linux and macOS. Note that Windows is not currently supported.
```

Reply:
done